fe/fe-core/src/main/java/com/starrocks/catalog/ScalarFunction.java

```diff
@@ -67,6 +67,11 @@ public class ScalarFunction extends Function {
     private String prepareFnSymbol;
     @SerializedName(value = "closeFnSymbol")
     private String closeFnSymbol;
+    // isolated/shared
+    private static final String ISOLATED = "isolated";
+    private static final String SHARED = "shared";
+    @SerializedName(value = "isolation")
```

Comment:
use isolation_mode? or just use is_isolated?

Reply:
isolation will be OK